Makefile

```diff
@@ -62,12 +74,20 @@ foreman-proxy-selinux-disable: common/selinux-disable.sh
 	bash $< "foreman-proxy" > $@
 
 %.pp: %.te
-	-mkdir ${TMPDIR} || rm -rf ${TMPDIR}/*
-	cp $< ${<:.te=.fc} $< ${<:.te=.if} ${TMPDIR}/
+	-rm -rf ${TMPDIR} 2>/dev/null
+	-mkdir ${TMPDIR} 2>/dev/null
+	cp $< ${<:.te=.fc} ${<:.te=.if} ${TMPDIR}/
 	sed -i 's/@@VERSION@@/${VERSION}/' ${TMPDIR}/*.te
-	make -C ${TMPDIR} -f /usr/share/selinux/devel/Makefile NAME=${VARIANT} DISTRO=$(DISTRO)
+	$(MAKE) -C ${TMPDIR} -f /usr/share/selinux/devel/Makefile NAME=${VARIANT} DISTRO=$(DISTRO)
 	mv ${TMPDIR}/$@ .
 
+%.pp.tmp: %.pp
+	$(info ************ LOADING MODULE $< ************)
+	semodule -i $<
+	touch $@
```

Comment:
These .pp.tmp files should be in .gitignore (perhaps ignore .pp.* instead of specifics), and consider calling them .load or something as they're just markers?